app/state_machines/pageflow/media_encoding_state_machine.rb

```diff
@@ -19,7 +19,7 @@ module EncodedFileStateMachine
           transition any => 'waiting_for_encoding'
         end
 
-        event :retry do
+        event :can_retry_encoding do
```

Comment:
Since the state machine already defines `can_retry_encoding?`, this difference in method names is a little to subtle for my taste.  Maybe just `retry_encoding`?